src/index.html

```diff
@@ -7,6 +7,350 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <!-- #region header -->
+      <header class="header">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+          class="header__logotype"
+          >
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#Apple"
+                class="nav__link is-active"
+              >
```

Comment:
```suggestion
              <a href="#Apple" class="nav__link is-active">
```

If you have one or two attributes you can leave them in one line